www/main/common/sttm-ui/search-results/SearchResults.jsx

```diff
@@ -107,6 +109,7 @@ const SearchResults = ({
             )} ${ang} `}</span>
           )}
           <span className="gurmukhi">{highlightKeywords(verse, searchQuery)}</span>
+          {currentLanguage === 'en' && <div className="eng-verse">{anvaad.translit(verse)}</div>}
```

Comment:
veerji, recently I learned this. We should keep our static strings in a standard file. Just like 'en' all the strings from all the files should be moved to a javascript file. You can create this issue and work on this later.